compiler/rustc_data_structures/src/flock/linux.rs

```diff
@@ -18,7 +18,7 @@ impl Lock {
             .read(true)
             .write(true)
             .create(create)
-            .mode(libc::S_IRWXU as u32)
+            .mode((libc::S_IRUSR | libc::S_IWUSR) as u32)
```

Comment:
```suggestion
            .mode(0o600)
```

These constants are almost always less readable than the numeric permissions.